source/scripting_v3/GTA/World.cs

```diff
@@ -1063,6 +1073,23 @@ public static InteriorProxy GetClosest(Vector2 position, params InteriorProxy[]
 			return closest;
 		}
 
+		/// <summary>
+		/// Sets the density of all ambient vehicles in the world.
+		/// Must be called each frame to remain in effect.
+		/// </summary>
+		/// <param name="density">The density multiplier, between 0 and 1.</param>
+		public static void SetAmbientVehicleDensityMultiplierThisFrame(float density)
+		{
+			Function.Call(Hash.SET_VEHICLE_DENSITY_MULTIPLIER_THIS_FRAME, density);
+		}
+
+		/// <summary>
+		/// Sets the density of ambient peds in the world.
+		/// Must be called each frame to remain in effect.
+		/// </summary>
+		/// <param name="density">The density multiplier, between 0 and 1.</param>
+		public static void SetAmbientPedDensityMultiplierThisFrame(float density) => Function.Call(Hash.SET_PED_DENSITY_MULTIPLIER_THIS_FRAME, density);
+
```

Comment:
I think the param name would be better if named like `densityMult` (there's no word that indicates "multiplier" yet).

Also, it could be better if we indicate that the methods will ignore the value and do nothing if the passed value is not in the range of 0 to 1 inclusive? Too bad R* didn't let us set values more than 1.0 via density natives, while the game code doesn't care the value range for densities and you can directly write values to make the game spawn more traffic and more parked vehicles, like in 3D Era games. I'm not planning to search for memory patterns for density variables soon though.